mobile/adaptation.html

```diff
@@ -1,50 +1,52 @@
-        <p>The <a data-featureid="viewport-units">viewport-relative CSS units <code>vw</code> and <code>vh</code></a> let design layouts that adapt to the dimensions of the viewport, representing a percentage of the current viewport dimensions.</p>
-      </div>
-      <div data-feature="Responsive images">
-      <p>The <a href="https://www.w3.org/community/respimg/">Responsive Images Community Group (RICG)</a> developed an extension to HTML, known as the <a data-featureid="picture"><code>picture</code> element</a>, that allows authors define what image to load depending on device capabilities and/or other media features.</p>
-      <p>As a complementary approach, the <a data-featureid="srcset"><code>srcset</code> attribute</a>, specified by the <a href="https://www.w3.org/community/whatwg/">WHATWG</a> and also published as an extension to HTML, let Web developers define the various device pixel ratios of an image, letting the browser pick the best choice for the pixel density of the screen.</p>
-      <p><a data-featureid="svg12">SVG</a>, which lets define images that can be scaled up and down without any loss of quality, is another critical tool to the development of Web applications that adapt to the resolution of the underlying device.</p>
-      </div>
-        </section>
-
-        <section class="featureset in-progress">
-          <h2>Specifications in progress</h2>
-          <p data-feature="Media Capture Capabilities">The <a data-featureid="getusermedia-cap">Media Capture and Streams</a> API exposes some specific information on capabilities of camera and microphones to make it possible to take advantage of the large variety of media capturing devices provided on mobile phones.</p>
-
-          <div data-feature="CSS-based adaptation">
+        <h2>Well-deployed technologies</h2>
+        <div data-feature="CSS-based adaptation">
+          <p><a data-featureid="mediaqueries">CSS Media Queries</a> defines a mechanism that allows adapting the layout and behavior of a Web page based on some of the characteristics of the device, most notably based on the screen resolution.</p>
+          <p>The <a data-featureid="viewport-units">viewport-relative CSS units <code>vw</code> and <code>vh</code></a> let design layouts that adapt to the dimensions of the viewport, representing a percentage of the current viewport dimensions.</p>
```

Comment:
Do we need to mention `vmin` and `vmax`, too? Or we just remove the specific units, and only use "viewport-percentage lengths" or "viewport-relative CSS units" here?

Reply:
That may be too technical for this document, but at the same time, mentioning these additional units does not take much space, so why not. See updated file.

I also improved the description a little bit.